### ISSUE TITLE
Compatibility Issue

### ISSUE THREAD

**Issue by dannyy3566** [dustine/ChargeTransmission#2]
Using this mod and [Atomic Robots by FunCarcass](https://mods.factorio.com/mods/FunCarcass/AtomicBots/discussion) on the mod portal results in an error message and a crash to the main menu.
The error message reads;

```
Error while running event
ChargeTransmission::on_tick(ID 0)
__ChargeTransmission__/control.lua:433:
attempt to perform arithmetic on field '?' (a nil value)
```

**Comment by dustine** [comment]
Thank you for the bug report! I'm really confused by this as Atomic Robots was one mod I specifically wrote compatibility code for (as there's no point charging never-stopping robots! Not that you could anyhow, they don't have any energy buffer to charge). 

Seems it must have broken, on my side, when I updated to 0.4 and I never noticed it. I'll see what's up and make sure it's added to my testing suit again so this doesn't happen in the future.

**Comment by dustine** [comment]
Yep. Fixed in 0.4.4, I forgot you can't multiply nils with numbers.

**Comment by dannyy3566** [comment]
Good to hear, and thanks for the fast response!
